docs/Hosts/Developer-Workstations.md

```diff
@@ -46,16 +46,15 @@ checked out and running on workstations, though this feature is not
 necessarily dependent on workstations, and could be used on local
 systems.
 
-Project configuration include a "Workstation Setup" section where
-administrators declare a number of simple commands (and directory
-contexts) that will run a project's setup. These commands are *not*
-shell interpolated, and are *not* meant to provision the development
-environment (e.g. install system packages or modify configuration
-files in `~/`). Use these commands to clone a repository, generate
-index files, and/or run a test build.
+Project configuration include a ["Workstation Setup" section](../Project-Configuration/Project-and-Distro-Settings.md#virtual-workstation-commands)
```

Comment:
includes**

Reply:
also might suggest saying "The project page" since we often rfer to the yaml as the project configuration (which is non ideal but)